src/pages/ProfilePage/index.js

```diff
@@ -9,9 +9,9 @@ function ProfilePage() {
   return (
     <Container className={classes.root}>
       <ProfileHero />
-      <Typography className={classes.title} variant="h2" color="textPrimary">
+      {/* <Typography className={classes.title} variant="h2" color="textPrimary">
         Profile & settings
-      </Typography>
+      </Typography> */}
```

Comment:
@rajatgupta24  please remove this comment! and related CSS implementation if there. and @rajatgupta24  small suggestion if you make any changes in any component please check for its responsiveness for all screens.

Reply:
Hi @adarsh-technocrat, should I add the "profile & setting" heading again, I removed it.

Reply:
> Hi @adarsh-technocrat, should I add the "profile & setting" heading again, I removed it.

@rajatgupta24 well it's looking good remove that!

Reply:
Hey, @rajatgupta24  keep this "profile & setting" I have made few changes to it !!